mlrun/api/db/sqldb/models.py

```diff
@@ -161,14 +161,20 @@ class Run(Base, HasStruct):
 
     class Schedule(Base):
         __tablename__ = "schedules_v2"
-        project = Column(String, primary_key=True)
-        name = Column(String, primary_key=True)
+        __table_args__ = (UniqueConstraint("project", "name", name="_schedules_v2_uc"),)
+
+        Label = make_label(__tablename__)
+
+        id = Column(Integer, primary_key=True)
```

Comment:
Why do we need the ID ? didn't in our phone call we said it isn't really needed ?

Reply:
Also if we do need it, who fills this one ? what will happen to old schedules that didn't had id on upgrade ?

Reply:
When I tried it the other way, having 2 foreign keys created problems with reverse searching and it didn’t work properly. So we do need it. 
It gets filled automatically with a sequential id(this is an bd feature in most databases, not sqlalchemy) - you can see that in the test I wrote.